vaxrank/cli.py

```diff
@@ -190,38 +197,108 @@ def load_template_data(args):
         min_epitope_score=args.min_epitope_score)
 
     ranked_list_for_report = ranked_list[:args.max_mutations_in_report]
-    df = dataframe_from_ranked_list(ranked_list_for_report)
-    logger.debug(df)
+    data = {
+        'variants': ranked_list_for_report,
+        'saved_args': args,
+    }
+    logger.info('About to save args: %s', data['saved_args'])
+
+    # save pickle data if necessary. as of time of writing, vaxrank takes ~25 min to run,
+    # most of which is core logic. the formatting is super fast, and it can
+    # be useful to save the data to be able to iterate just on the formatting
+    if args.output_pickle_file:
+        with open(args.output_pickle_file, 'wb') as f:
+            pickle.dump(data, f)
```

Comment:
Any explicit value other than 0 fails. Here is what happens at deserialization, when I dump with protocol=2:

```
  File "/Users/julia/code/vaxrank/vaxrank/cli.py", line 186, in ranked_variant_list_with_saved_args
    data = cPickle.load(f)
TypeError: __new__() takes at most 4 arguments (6 given)
```
Would like to leave it unspecified for now - sadly, can't test on py3 right now because FMIndex is busted on py3 so it errors for other reasons.